core-bundle/src/Resources/config/services.yml

```diff
@@ -28,6 +28,22 @@ services:
             - { name: container.service_subscriber }
         public: true
 
+    Contao\CoreBundle\Controller\FaviconController:
+        arguments:
+            - '@contao.framework'
+            - '@?fos_http_cache.http.symfony_response_tagger'
+        tags:
+            - controller.service_arguments
```

Comment:
The `controller.service_arguments` tag is not needed if you don't get services in your action methods.

Reply:
afaik it is still needed to dependency injection in the constructor.

Reply:
No, what you mean is the service locator. But this controller does not extend from `AbstractController` so service locator is not used at all.

Reply:
My custom controllers do not extend from `AbstractController` either and without the `controller.service_arguments`, no services would be injected into the constructor of the controller.

Reply:
Hm, or not, just checked it again. Setting the controller to `public: true` is enough.

Reply:
Guys, this PR is merged. If you want changes, create a new PR. Nothing will happen here.

Reply:
I'm still following the discussion though.

Reply:
The `__invoke()` method needs the `Request` object. Will this still work without the tag?